lib/tapioca/cli.rb

```diff
@@ -77,7 +77,52 @@ def dsl(*constants)
       end
     end
 
-    desc "generate [gem...]", "generate RBIs from gems"
+    desc "gem [gem...]", "generate RBIs from gems"
+    option :all,
```

Comment:
In my opinion, it would be more intuitive for the default gem command be to generate all and `sync` be an option. E.g.:
```shell
bin/tapioca gem # generates everything
bin/tapioca gem rails # generates specific gem
bin/tapioca gem --update # (or --sync) updates for all gems
bin/tapioca gem rails --update # (or --sync) updates for a specific gem
```
To my taste, this feels more consistent. In the way it's currently implemented, the gem command can either generate or sync, depending on whether we pass a gem name as an argument.

Thoughts?

Reply:
I've been pushing for the default action to be `sync`, since that works properly ~90% of the time and implies huge time-saving benefits. For example, on Core, if everyone were to regenerate the whole gem RBI set, everybody would be waiting for 5-6 mins while all random gem RBIs are generated.

That's why I've made `sync` the default action. If you give specific gem names, then those gems get generated and if you supply `--all` then all gems get generated. There is no case of syncing specific gems anyway, since `sync` is meant to bring things in line with Gemfile versions.